components/proxy/conf/Caddyfile

```diff
@@ -210,8 +210,21 @@ https://{$GITPOD_DOMAIN} {
 	handle @backend_wss {
 		gitpod.sec_websocket_key
 
+		@slow {
```

Comment:
:orange_heart:  Love this change - simply on-point :smile:

I'd only suggest we user a custom header instead of the websocket one. Also, we should check this on other HTTP routes that are sent against server as well :point_down: .